### PR TITLE
Core/adding Parameters serialization

### PR DESCRIPTION
**Pull request by RiccardoRossi**
this exports Load and Save methods for the serialization of Parameters

### CODE REVIEW

**Review by loumalouomega — APPROVED** [review]
OK for me

**Comment by loumalouomega** [comment]
Added to new parameters

File: kratos/tests/test_kratos_parameters.py [hunk 1]
```diff
@@ -612,6 +612,24 @@ def test_double_vs_null_validation(self):
...
+
+        file_name = "parameters_serialization"
+        serializer_flag = SerializerTraceType.SERIALIZER_NO_TRACE
+        Serializer(file_name, serializer_flag).Save("ParametersSerialization",tmp)
+        tmp = 0
+
+        loaded_parameters = Parameters()
+        Serializer(file_name, serializer_flag).Load("ParametersSerialization",loaded_parameters)
+
+        self.assertEqual(check, loaded_parameters.WriteJsonString())
```
Comment (philbucher) [inline kratos/tests/test_kratos_parameters.py:627]: is this check sufficient?

Wouldn't it be better to also fill the parameters with some stuff and check that after loading?

Comment (RiccardoRossi) [reply kratos/tests/test_kratos_parameters.py:627]: Hi @philbucher  in principle what you are saying is exactly what i am doing. The test is doing the following:

1 - I am filling a Parameter objects with "defaults" (it was the most complete of the example jsons at the beginning of the file).
2 - I am creating an auxiliary string by using WriteJsonString on such parameters object
3 - i am serializing the Parameters object and i delete it
4 - I create an empty object and fill it by loading the serialization data
5 - i check that if i call WriteJsonString on this loaded object i obtain exactly the same as for the original object...
6 - i delete the .rest file to leave the directory clean

i think this is a quite complete test...

**Pull request merged**